.github/workflows/tests.yml

```diff
@@ -86,4 +88,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-        run: npx semantic-release
+        run: echo npx semantic-release
```

Comment:
This will prevent semantic-release from running, but that is ok initially. I will do some testing to make sure this actually works before we release it.

Reply:
yeah -- that's what I was thinking the purpose of that change was.